build/references/coreth-arc20s.md

```diff
@@ -0,0 +1,103 @@
+# Coreth ANTs and ARC20s
+
+## ANTs on the C-Chain
+
+### Why move an ANT from the X-Chain to the C-Chain?
+
+An Avalanche Native Token (ANT) is any token created on the X-Chain. These tokens can be exchanged on the X-Chain at lightning fast speeds taking advantage of the superior performance offered by using a DAG instead of a strictly linear chain.
+
+When we want to use smart contract functionality, we need a total ordering of state transitions (transactions), so that each transaction can modify the state of the network without having to worry about external dependencies.
+
+Therefore, when we want to use smart contract functionality, we must perform an atomic swap to move ANTs from the X-Chain to the C-Chain.
+
+### ANTs on the C-Chain
+
+#### Native Coin - AVAX
+
+ANTs on the C-Chain are either AVAX &emdash; the Avalanche native asset &emdash; or another asset that was created on the X-Chain.
+
+The C-Chain is a modified instance of the EVM, so AVAX on the C-Chain is equivalent to ETH on the Ethereum network. When you create or call a smart contract, you pay for gas costs with AVAX. You can natively transfer AVAX between accounts, send it to a smart contract, all using native EVM tools and libraries.
+
+#### EVM Transaction
+
+An EVM Transaction is composed of the following fields:
+
+* **`nonce`** Scalar value equal to the number of transactions send by the sender
+* **`gasPrice`** Scalar value equal to tthe number of Wei paid per unit of gas to execute this transaction
+* **`gasLimit`** Scalar value equal to the maximum amount of gas that should be used in executing this transaction
+* **`to`** The 20 byte address of the message call's recipient. If the transaction is creating a contract, `to` is left empty
+* **`value`** Scalar value of native asset (AVAX) measured in wei to be transferred to the message call's recipient or in the case of a contract creation, as an endowment to the newly created contract.
+* **`v, r, s`** Values corresponding to the signature of the transaction.
+* **`data`** Unlimited size byte array specifying the input data to a contract call or (if creating a contract) the EVM bytecode for the account initialisation process
+
+#### Native Assets - Tokens
+
+Non-AVAX Assets, however, have no counterpart within the EVM. Therefore, Coreth makes minor modifications to the EVM and state management in order to support asset balances and the ability to transfer these assets on the C-Chain.
```

Comment:
I'd rephrase to `Non-AVAX Assets, however, have no native representation on the EVM.`